server/views/pages/contacts/manage/confirmDeletePhone.njk

```diff
@@ -0,0 +1,79 @@
+{% extends "../../../partials/layout.njk" %}
+{% from "../../../partials/miniProfile/macro.njk" import miniProfile %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
```

Comment:
I think these two can be removed , not used? 
`{% from "govuk/components/input/macro.njk" import govukInput %}
{% from "govuk/components/select/macro.njk" import govukSelect %}`

Reply:
will tidy